docs/api/web-contents.md

```diff
@@ -696,7 +696,7 @@ Executes the editing command `cut` in web page.
 
 Executes the editing command `copy` in web page.
 
-### `contents.copyImageAt(x, y)`
+#### `contents.copyImageAt(x, y)`
```

Comment:
Yeah this was tricky to find. I discovered it because a test was failing on a suddenly-missing method farther down in the list.